### PR TITLE
DIV-5909: Using common lib instead cfs micro-service

### PR DESCRIPTION
**Pull request by yarpo**
# Description

Replacing `CaseFormatterClient` with `CaseFormatterService` from div-common-lib 

Fixes # (issue)
https://tools.hmcts.net/jira/browse/DIV-5909

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -77,6 +77,70 @@ case:
       api:
         baseurl: ${CASE_FORMATTER_SERVICE_API_BASEURL:http://localhost:4011}
 
+cohort: onlineSubmissionPrivateBeta
+
+document:
```
Comment (yarpo) [inline src/main/resources/application.yml]: This is needed by common-lib :(

File: src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/ObjectMapperConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.divorce.casemaintenanceservice.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ObjectMapperConfiguration {
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/ObjectMapperConfiguration.java]: Removed?

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/ObjectMapperConfiguration.java]: you're right

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -77,6 +77,70 @@ case:
...
+  management:
+    store:
+      url: ${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:5006}
+
+court:
```
Comment (alexwakeman) [inline src/main/resources/application.yml]: potentially we could adapt this to use the Spring Config handler, avoiding a big JSON string and using YAML proper. Example of which is in COS.

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -77,6 +77,70 @@ case:
...
+      \"southWest\":{
+        \"divorceCentre\":\"South West Regional Divorce Centre\",
+        \"courtCity\":\"Southampton\",
+        \"poBox\":\"PO Box 1792\",
+        \"postCode\":\"SO15 9GG\",
+        \"openingHours\":\"Telephone Enquiries from: 8.30am to 5pm\",
+        \"email\":\"sw-region-divorce@hmcts.gsi.gov.uk\",
+        \"phoneNumber\":\"0300 303 0642\",
+        \"siteId\":\"AA03\"
+      },
```
Comment (alexwakeman) [inline src/main/resources/application.yml]: 😢

**Pull request closed without merging**